python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -533,13 +535,14 @@ def aten〇max〇dim(self: List[int], dim: int, keepdim: bool = False) -> Tuple[
     return reduced_shape, reduced_shape
 
 def aten〇mean〇dim(self: List[int], dim: List[int], keepdim: bool = False, dtype: Optional[int] = None) -> List[int]:
+    if len(dim)==0:
+        dim = list(range(len(self)))
```

Comment:
This is fine for now, but you should upstream this fix to PyTorch as well

Reply:
Sure, will do.